### PR TITLE
Allow versions like 3.5.0.1 for extension debian packages

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

File: deb-package-builder/functions.sh [hunk 1]
```diff
@@ -47,7 +47,7 @@ download_from_pecl()
         with_retry "pecl download ${PECL_PACKAGE_NAME}" 6
         # determine the downloaded version
         EXT_VERSION=$(ls ${PACKAGE_SHORT_NAME}-*.tgz | \
-                sed "s/${PACKAGE_SHORT_NAME}-\([0-9][0-9]*\.[0-9][0-9]*\.[0-9][0-9]*\)\.tgz/\1/")
+                sed -E "s/${PACKAGE_SHORT_NAME}-([0-9]+.[0-9]+(.[0-9]+)+).tgz/\1/")
```
Comment (chingor13) [inline deb-package-builder/functions.sh]: Do we need to keep the escaping for the "." characters?

Comment (tmatsuo) [reply deb-package-builder/functions.sh]: Good catch, done

**Comment by tmatsuo** [comment]
Just in case I'm running the package builder locally. I'll merge once it builds correct packages

**Comment by tmatsuo** [comment]
Now it's failing on opencensus-0.1.0RC1.tgz

**Comment by chingor13** [comment]
Doh, I uploaded a devel version of opencensus to ensure the build works. 0.1.0 will be tagged alpha and will be downloadable via `pecl download opencensus-alpha`.

In any case, we should probably make the version string even more lenient.

**Comment by tmatsuo** [comment]
Yeah I'm thinking to allow any postfix string

**Comment by tmatsuo** [comment]
It builds correct packages including
- gcp-php72-opencensus_0.1.0RC1-7.2.0-2_amd64.deb
- gcp-php72-protobuf_3.5.0.1-7.2.0-2_amd64.deb

**Pull request merged**